modules/arch-template-service-broker.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * architecture/service-catalog.adoc
+
+[id='template-service-broker-{context}']
+= About the template service broker
+
+The _template service broker_ (TSB) gives the service catalog visibility into
```

Comment:
@kalexand-rh I'm pulling in this file as sb-about-template-service-broker.adoc with this PR [1] in case you want to eventually rebase and adjust to include this file instead. Should be merged in the next day or so.

[1] https://github.com/openshift/openshift-docs/pull/14067